### PR TITLE
Fix RTInfo for ReduceL2Decomposition

### PR DESCRIPTION
**Pull request by nkogteva**
### Details:
 - *Constant name is missing in fused names of new operations in ReduceL2Decomposition which lead to problem after subsequent call of  ConvertReduceToPooling transformation. Model facenet is failed on HETERO:NVIDIA,GPU device due to cut Constant.*
 
```
Unsupported parameter nodes type in ReduceL2_361 (ReduceL2)
```

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/inference/tests/unit/query_model_test.cpp [hunk 1]
```diff
@@ -461,3 +463,30 @@ TEST_F(GetSupportedNodesTest, ShuffleChannelFusion) {
...
+            m.register_pass<ov::pass::ReduceL2Decomposition>();
+            m.register_pass<ov::pass::ConvertReduceToPooling>();
+            m.run_passes(model);
+        },
+        [&](const std::shared_ptr<ngraph::Node>& op) {
+            // Pooling is supported, but Sqrt is not
+            return ov::op::util::is_parameter(op) || ov::op::util::is_output(op) || ov::op::util::is_constant(op) ||
+                   (std::dynamic_pointer_cast<ov::opset1::AvgPool>(op) != nullptr);
+        },
+        {});  // Check that constant axis is removed from supported
```
Comment (ilya-lavrenov) [inline src/inference/tests/unit/query_model_test.cpp:491]: again, why do we need a test for QN?
The bug in transformations and we need a test in `transformations` folder.

Comment (nkogteva) [reply src/inference/tests/unit/query_model_test.cpp:491]: > again, why do we need a test for QN? The bug in transformations and we need a test in `transformations` folder.

Transformation tests check that RT info is located inside operations. And it is actually located. The question what exactly located inside. This bug is reproduced in combinations of two sunsequent transformations and in condition when only part of new transformed graph is supported by plugin. Any improvements for tranformations implementatios and tests are welcome, but it should not be part of bugfix.

Comment (ilya-lavrenov) [reply src/inference/tests/unit/query_model_test.cpp:491]: > The question what exactly located inside

@itikhono can we improve tests for transformations to check rt_info more strictly?

Comment (itikhono) [reply src/inference/tests/unit/query_model_test.cpp:491]: @t-jankowski could you take a look?

Comment (t-jankowski) [reply src/inference/tests/unit/query_model_test.cpp:491]: As a hot fix it's ok to place it here (considering suggestion I put in the other comment). However actual solution should be either in `ngraph::copy_runtime_info` or transformations dedicated wrapper function.
As of the transformations copying runtime info test - yes, it should be improved.

**Review by t-jankowski — APPROVED** [review]
lgtm as of transformations part

**Pull request merged**